e2e/src/QRScanner.spec.js

```diff
@@ -3,7 +3,7 @@ import { quickOnboarding, waitForElementId } from './utils/utils'
 
 // Re-enable for android when react-native-reanimated is updated to v3
```

Comment:
do we still need this comment?

Reply:
Removed in 9c9b201a436e4deaeb4d5d3cd9d313db799661b8!